src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -123,6 +123,10 @@ struct CPUStreamsExecutor::Impl {
         }
 
 #if OV_THREAD == OV_THREAD_TBB || OV_THREAD == OV_THREAD_TBB_AUTO
+#    ifndef TBBBIND_2_5_AVAILABLE
+#        define TBBBIND_2_5_AVAILABLE 0
+#    endif
+#    define USE_TBBBIND_2_5 TBBBIND_2_5_AVAILABLE
```

Comment:
USE_TBBBIND_2_5 is defined in parallel_custom_arena.cpp. Whether it can be reused?